docs/hpc/containers.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Containers
+parent: HPC
+nav_order: 4
+---
+
+### Contents
+1. Overview
+2. Default Lab Container
+3. Building a New Container
```

Comment:
Currently working on this section. Will integrate changes into a branch once PR is integrated.